packages/view-types/obs-sets-manager/src/SetsManagerButtons.js

```diff
@@ -129,27 +133,27 @@ export function SetOperationButtons(props) {
         <>
           <button
             onClick={onUnion}
-            title="New set from union of checked sets"
+            title={unionText}
             type="submit"
             disabled={!hasCheckedSetsToUnion}
           >
-            <SetUnionSVG />
+            <SetUnionSVG aria-label={`Icon to select a ${unionText}`} />
```

Comment:
Perhaps we could set the SVGs icons to be `aria-disabled` and add the labels to the buttons instead?